pkg/aws/tags/client_create.go

```diff
@@ -0,0 +1,61 @@
+package tags
+
+import (
+	"context"
+	"sort"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2Types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/giantswarm/microerror"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/giantswarm/aws-vpc-operator/pkg/errors"
+)
+
+type CreateTagsInput struct {
+	RoleARN    string
+	ResourceId string
+	Tags       map[string]string
+}
+
+func (c *client) Create(ctx context.Context, input CreateTagsInput) error {
+	logger := log.FromContext(ctx)
+	logger.Info("Started creating tags")
+	defer logger.Info("Finished creating tags")
+
+	if input.RoleARN == "" {
+		return microerror.Maskf(errors.InvalidConfigError, "%T.RoleARN must not be empty", input)
+	}
+	if input.ResourceId == "" {
+		return microerror.Maskf(errors.InvalidConfigError, "%T.ResourceId must not be empty", input)
+	}
+
+	// For testing, we need sorted keys
+	sortedKeys := make([]string, 0, len(input.Tags))
+	for k := range input.Tags {
+		sortedKeys = append(sortedKeys, k)
+	}
+	sort.Strings(sortedKeys)
```

Comment:
Where's this needed?

Reply:
Taken from CAPA, needed for tests there for some reason. I did not check why exactly, wanted to check that part when I add tests.

Reply:
If we've no tests yet then lets remove it. It just adds noise :) We can add it in if and when its needed.